src/graphql/root/mutation/ln-invoice-fee-probe.ts

```diff
@@ -34,13 +36,28 @@ const LnInvoiceFeeProbeMutation = GT.Field({
     const btcWalletValidated = await validateIsBtcWalletForMutation(walletId)
     if (btcWalletValidated != true) return btcWalletValidated
 
-    const feeSatAmount = await Payments.getLightningFeeEstimation({
+    const { result: feeSatAmount, error } = await Payments.getLightningFeeEstimation({
       walletId,
       paymentRequest,
     })
-    if (feeSatAmount instanceof Error) {
-      const appErr = mapError(feeSatAmount)
-      return { errors: [{ message: appErr.message }] }
+
+    if (feeSatAmount && error instanceof Error) {
+      return {
+        errors: [{ message: mapError(error).message }],
+        ...normalizePaymentAmount(feeSatAmount),
+      }
+    }
+
+    if (error instanceof Error) {
+      return {
+        errors: [{ message: mapError(error).message }],
+      }
+    }
+
+    if (feeSatAmount === null) {
```

Comment:
in this point you can use `if (!feeSatAmount`

Reply:
:thinking: 0 is also a valid value so we can get line 63.. so maybe this change is not required if you are sure that undefined is not a valid valude for `feeSatAmount`

Reply:
Should only get `null` values at this point for the `.err` case ([here](https://github.com/GaloyMoney/galoy/blob/2ad1941090bea25dc152e43c21d1b1a887123c05/src/app/partial-result.ts#L12))